### ISSUE TITLE
Integrate with pkg.go.dev

### ISSUE THREAD

**Issue by marwan-at-work** [gomods/athens#1546]
Now that pkg.go.dev will be open sourced (more here https://groups.google.com/forum/#!msg/golang-dev/mfiPCtJ1BGU/OZ9cU3SgBgAJ) 

This is a place holder ticket to see if/how we can integrate Athens with it :v:

**Comment by arschles** [comment]
One idea - we could add it to some of our deployment artifacts. For example, you could install the helm chart and it would optionally have a doc server all set up alongside your Athens server.

Obviously depends on if the server is easily packaged up in an image and easily deployed

**Comment by twexler** [comment]
This feels a bit out of scope for Athens, but could be part of a larger initiative to provide non-Google-hosted Go development resources

**Comment by arschles** [comment]
Agreed. If `pkg.go.dev` is like moddoc, you would be able to point it at Athens and it would "just work".

If that is not the case, I hope there would be a way we can make them work together. I would advocate we build the software to do that in a different repo. Or at least in a different binary.

**Comment by twexler** [comment]
I wasn't familiar with `moddoc`, nice work @marwan-at-work! 

That would be the most ideal. It's a bit disappointing `go doc`'s http server was deprecated, as having access to the on-disk cache that Athens uses could also be a great excuse to run a docs server with it.
